src/library/scala/util/Using.scala

```diff
@@ -0,0 +1,287 @@
+package scala.util
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import scala.util.control.{ControlThrowable, NonFatal}
+
+/** A utility for performing automatic resource management. It can be used to perform an
+  * operation using resources, after which it will release the resources, in reverse order
+  * of their creation. The resource opening, operation, and resource releasing are wrapped
+  * in a `Try`.
+  *
+  * If more than one exception is thrown by the operation and releasing resources,
+  * the exception thrown ''first'' is returned within the `Try`, with the other exceptions
+  * [[java.lang.Throwable.addSuppressed(Throwable) added as suppressed exceptions]]
```

Comment:
meandering `]]`
```scala
* [[java.lang.Throwable.addSuppressed(Throwable)]] added as suppressed exceptions
```

Reply:
no, it's correct. I want the text of the link to say `added as suppressed exceptions`, and to have it link to `java.lang.Throwable.addSuppressed(Throwable)`